Sources/Turf/CoreLocation.swift

```diff
@@ -188,12 +188,13 @@ extension LocationCoordinate2D: Equatable {
     
     /// Returns the direction from the receiver to the given coordinate.
     public func direction(to coordinate: LocationCoordinate2D) -> LocationDirection {
-        return RadianCoordinate2D(self).direction(to: RadianCoordinate2D(coordinate)).toDegrees()
+        return RadianCoordinate2D(self).direction(to: RadianCoordinate2D(coordinate)).converted(to: .degrees).value
     }
     
     /// Returns a coordinate a certain Haversine distance away in the given direction.
     public func coordinate(at distance: LocationDistance, facing direction: LocationDirection) -> LocationCoordinate2D {
-        let radianCoordinate = RadianCoordinate2D(self).coordinate(at: distance / metersPerRadian, facing: direction.toRadians())
+        let radiansDirection = Measurement(value: direction, unit: UnitAngle.degrees)
```

Comment:
It wouldn’t be strictly necessary, but it could be nice to offer a `LocationCoordinate2D.coordinate(at:facing:)` that takes a `Measurement<UnitAngle>` just like the one on `RadianCoordinate2D`. Then this method becomes a lightweight wrapper around it that just converts the `LocationDirection` to a degree measurement. But that’d only be for completeness’ sake.

Reply:
done!